webapp/main/styles/buttons.less

```diff
@@ -82,6 +82,17 @@ button:hover {
   }
 }
 
+.btn-checkbox {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: 400;
+
+  .fra-checkbox {
+    margin-right: 8px;
+  }
+}
```

Comment:
should this go to webapp/components/buttonCheckBox/buttonCheckBox.less instead ?